src/sql/workbench/api/common/notebooks/vscodeExecuteProvider.ts

```diff
@@ -86,6 +86,34 @@ class VSCodeKernel implements azdata.nb.IKernel {
 			this._kernelSpec.supportedLanguages = this._controller.supportedLanguages;
 		}
 
+		// Store external kernel names for .NET Interactive kernels for when notebook gets saved, so that notebook is usable outside of ADS
+		if (this._kernelSpec.name === 'jupyter-notebook' && this._kernelSpec.display_name === DotnetInteractiveDisplayName) {
```

Comment:
It would be good to add the further description and samples you have in this PR to the comment so others looking at this have more context about what these changes are doing.